mergify_engine/tests/unit/test_crypto.py

```diff
@@ -0,0 +1,23 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2020 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from mergify_engine import crypto
+
+
+def test_encrypt():
+    x = "this is an amazing string, right? 🙄".encode()
+    assert crypto.decrypt(crypto.encrypt(x))
+    assert crypto.decrypt(crypto.encrypt(x))
```

Comment:
Duplicate line

Reply:
This was actually voluntary to test there was no side effect on the crypto module itself.